phpstan.neon

```diff
@@ -0,0 +1,13 @@
+parameters:
+    level: 6
```

Comment:
Can you try setting this to `max`? If the result seems unreasonable, try going down from 8 to 7 and eventually fall back to 6.